patches/minecraft/net/minecraft/world/gen/layer/HillsLayer.java.patch

```diff
@@ -0,0 +1,13 @@
+--- a/net/minecraft/world/gen/layer/HillsLayer.java
++++ b/net/minecraft/world/gen/layer/HillsLayer.java
+@@ -59,7 +59,9 @@
+ 
+       if (p_215723_1_.func_202696_a(3) == 0 || k == 0) {
+          int l = i;
+-         if (i == field_202799_f) {
++         Biome hill = Registry.field_212624_m.func_148745_a(i).getHill(p_215723_1_);
```

Comment:
`getByValue` can be null, so this should check for `null` before the getHill call as well, like this:
```
Biome biome = Registry.BIOME.getByValue(i);
Biome hill = biome == null ? null : biome.getHill(p_215723_1_);
```